.github/CODEOWNERS

```diff
@@ -177,3 +177,9 @@
 /x-pack/plugins/searchprofiler/  @elastic/es-ui
 /x-pack/legacy/plugins/snapshot_restore/  @elastic/es-ui
 /x-pack/plugins/watcher/  @elastic/es-ui
+
+# Endpoint
+/x-pack/plugins/endpoint/ @elastic/endpoint-app-team
```

Comment:
@jonathan-buttner I think the email alias is actually a Group under elastic (and not the repo). I took a look at the group called `endpoint-app-team` [here](https://github.com/orgs/elastic/teams/endpoint-app-team/members?utf8=%E2%9C%93&query=membership%3Achild-team) and there are 30 people on it, including non-dev (which I think is ok?), but it also seems to be missing some.

Reply:
Oh yeah I should have made that more clear, I think it's supposed to reference the team not the repo and yeah you linked the correct team. Yeah I think it's ok that it includes people that are not devs. If it's missing some people @crowens or the individual team leads can add them to the child teams (e.g. [endpoint-management](https://github.com/orgs/elastic/teams/endpoint-management)) or if need be we can create a new team under `endpoint-app-team`.